__tests__/integration/utils/test-utils-integration.js

```diff
@@ -0,0 +1,229 @@
+    addresses: [
+      'WTjhJXzQJETVx7BVXdyZmvk396DRRsubdw',
+      'Wdf7xQtKDNefhd6KTS68Vna1u4wUAyHjLQ',
+      'WaQf5igKpbdNyxTBzc3Nv8a8n4DRkcbpmX',
+    ]
+  },
+};
+
+export const HATHOR_TOKEN_ID = '00';
+
+/**
+ * Generates a random positive integer between the maximum and minimum values,
+ * with the default minimum equals zero
+ * @param {number} max
+ * @param {number} [min=0]
+ * @returns {number} Random number
+ */
+export function getRandomInt(max, min = 0) {
+  const _min = Math.ceil(min);
+  const _max = Math.floor(max);
+  return Math.floor(Math.random() * (_max - _min + 1)) + _min;
+}
+
+export class TestUtils {
+  /**
+   * Simple way to wait asynchronously before continuing the funcion. Does not block the JS thread.
+   * @param {number} ms Amount of milliseconds to delay
+   * @returns {Promise<unknown>}
+   */
+  static async delay(ms) {
+    return new Promise(resolve => {
+      setTimeout(resolve, ms);
+    });
+  }
+
+  /**
+   * Returns the Supertest `request` object for this application
+   * @returns {Test}
+   */
+  static get request() {
+    return request;
+  }
+
+  /**
+   * Generates seed words for a new wallet
+   * @returns {string|*}
+   */
+  static generateWords() {
+    return wallet.generateWalletWords();
+  }
+
+  /**
+   * Generates the header for requesting a protected route on the wallet headless
+   *
+   * @see TestUtils.stopWallet
+   * @param {string} walletId
+   * @returns {{"x-wallet-id"}}
+   */
+  static generateHeader(walletId) {
+    return { 'x-wallet-id': walletId };
+  }
+
+  /**
+   * Adds a message to the Transaction Log for a test
+   * @param message
+   */
+  static logTx(message) {
+    loggers.test.insertLineToLog(message)
+      .catch(err => console.error(err.stack));
+  }
+
+  /**
+   * Logs a custom error message to the correct output to be analyzed later
+   * @param {string} message Custom error message
+   */
+  static logError(message) {
+    console.error(message);
+  }
+
+  /**
+   * Starts a wallet. Prefer instantiating a WalletHelper instead.
+   * @param {WalletData} walletObj
+   * @returns {Promise<{start:unknown,status:unknown}>}
+   */
+  static async startWallet(walletObj) {
+    // Request the Wallet start
+    const response = await request
+      .post('/start')
+      .send({ seed: walletObj.words, 'wallet-id': walletObj.walletId });
+
+    // Handle errors
+    if (response.status !== 200) {
+      throw new Error(`Unable to start the wallet: ${walletObj.walletId}`);
+    }
+    if (!response.body.success) {
+      console.error(`Failure starting the wallet: ${response.body.message}`);
+      throw new Error(response.body.message);
+    }
+    const start = response.body;
+
+    // Wait until the wallet is actually started
+    let status;
+    while (true) {
+      const res = await request
+        .get('/wallet/status')
+        .set(TestUtils.generateHeader(walletObj.walletId));
+      if (res.body?.statusMessage === 'Ready') {
```

Comment:
Maybe it's better to validate the state and not the string.

`if (res.body?.statusCode === HathorWallet.READY)`

What do you think?

Reply:
Good to know this constant is available :+1: 
Refactored to use it.